tests/unittests/OperatorTest.cpp

```diff
@@ -135,6 +135,76 @@ TEST_P(InterpAndCPU, log) {
   }
 }
 
+TEST_P(Operator, logit) {
+  auto eps = 1E-6f;                // the default in Caffe2
+  constexpr std::size_t size = 10; // sample size for randomized tests
+
+  // known values: the logarithm of the odds for 25%, 50%, and 75% probabilities
+  auto *X = mod_.createPlaceholder(ElemKind::FloatTy, {1, 1, 3}, "X", false);
+  ctx_.allocate(X)->getHandle() = {0.25f, 0.5f, 0.75f};
+  auto *LogitX = F_->createLogit("logitX", X, eps);
+  auto *saveX = F_->createSave("saveX", LogitX);
+  auto *savePlaceholderX = saveX->getPlaceholder();
+  ctx_.allocate(savePlaceholderX);
+
+  // differential test:
+  // ensure we match an oracle `logit_test` (a C++ reimplementation test)
+  auto clamp_test = [](float v, float lo, float hi) {
+    return std::max(std::min(v, hi), lo);
+  };
+  auto logit_test = [clamp_test](float x, float eps = 1E-6f) {
+    float p = clamp_test(x, eps, 1.0f - eps);
```

Comment:
Consider changing this to `float p = std::max(std::min(x, 1.0f-eps), eps);`

`clamp_test` is only used here, so I don't see a reason to separate it out into its own lambda.